plugins/poll/assets/javascripts/widgets/discourse-poll.js.es6

```diff
@@ -894,6 +908,28 @@ export default createWidget("discourse-poll", {
     this.state.showResults = !this.state.showResults;
   },
 
+  removeVote() {
+    const { attrs, state } = this;
+    state.loading = true;
+    return ajax("/polls/vote", {
+      type: "PUT",
```

Comment:
Should probably use the `DELETE` verb and a different route on the server to handle the removal of a vote